docs/05-PipelineLaunchOptions.md

```diff
@@ -0,0 +1,99 @@
+# 5.0 - Pipeline Launch options
+
+Scans upload into LORIS and insertion pipeline's triggering can be done in a few 
+ways depending on each project's needs. The best choice will depend on whether 
+for example data is collected retrospectively or prospectively, and/or whether 
+the MRI protocol is harmonized across all the sites involved in the study.
+
+The different options available are graphically illustrated in the graph below.
+
+![PipelineLaunchOptions](images/PipelineLaunchOptions.png)
```

Comment:
Love the graph!!!

See picture below for a tiny comment on the graph
![pipelinelaunchoptions](https://user-images.githubusercontent.com/1402456/35337038-22369f58-00e8-11e8-9efc-34c6e7fb8283.png)

Reply:
yes, I did that in purpose to draw attention that this is useful in case of multiple uploads, but I can remove it (especially if it was not obvious to you why I did it), not a problem!